book/src/pages/foreword-1.md

```diff
@@ -19,8 +19,11 @@ Since we're assuming you have little programming experience we won't go into the
 Suffice to say there are different ways to think about and write computer programs, and we've chosen the functional programming approach.
 
 The reasons for choosing functional programming are more interesting.
-It's common to teach programming by what we call the "bag of syntax" approach.
-In this approach a programming language is taught a collection of syntactical features (variables, for loops, while loops, methods) and students are left to figure out on their own when to use each feature.
+It's common to teach programming by what we call the "bag of syntax" approach, where "syntax" is the term we use for the rules defining the structure of symbols in a programming language. 
+In English, we are taught that a sentence can be constructed using the rule: subject + verb + object e.g. "the artist paints a cat".
+A programming language might have a different set of rules and symbols to read `Artist.paint(cat);`. 
+This is a general example - languages have syntax variations, and it is worth noting that the English sentence is purely descriptive while the general code example is instructing the computer to paint a cat using 'Artist', which could be a library or object. 
```

Comment:
One of my favourite bits of pedantry: the em-dash.

You have used an [em-dash](https://www.merriam-webster.com/grammar/em-dash-en-dash-how-to-use) in the sentence "This is a general example&hellip;" but you have not done so correctly:

* em-dashes are matching delimiters like parentheses: they always come in pairs
* the em-dash is a different character (it's &mdash;) to a minus sign (which is -)
* there are no spaces around an em-dash

"This is a general example—languages have syntax variations—and it is worth noting&hellip;" is correct.